docker/docker-compose-refqa-3x.yml

```diff
@@ -53,7 +53,9 @@ services:
       MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-openmrs}
     volumes:
       - db-data:/var/lib/mysql
+    ports:
+      - 3306:3306
 
 volumes:
   openmrs-data: ~
-  db-data: ~
+  db-data: ~
```

Comment:
```suggestion
  db-data: ~

```

Reply:
btw, are there any changes we need to make to this file? (I mean after the changes on the distro repo?)

Reply:
There were some changes in the [docker-compose-distro.yml](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/3.x/docker-compose-distro.yml) file with the latest commits. So I update the file to fit it with the latest commits in [3.x](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/3.x) branch.

Reply:
Fixed EOL with [4dca1ad](https://github.com/openmrs/openmrs-test-3refapp/pull/3/commits/4dca1adc769581774ed8145ba09e1103ed87c4d7)